### PR TITLE
Milestone 8: Add forms

### PR DESCRIPTION
**Pull request by SaidRasinlic**
### In this milestone I've done the following things:

**Required Changes** ♻️  _(remove unused code) :_ Resolved removing all unused code (comments, etc) from the project ✔️

- Created a method called current_user in my ApplicationController to make current user data available to all controllers.
  - It will return the first user from the database.
- Created forms to perform the following functions:
  - Creates a Post on behalf of the current_user (used the method that I created in my ApplicationController).
  - Created a comment on behalf of the current_user (used the method that I created in my ApplicationController).
  - Allowed Users to add likes to Posts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ The Blog app will be a classic example of a blog website. I will create a fully
 
 ## Screenshot
 
-![Desktop Snapshot](./assets/images/Blog-App.png)
```
Comment (arslanbisharat) [inline README.md:8]: - [ ] please make sure there is no commenter code present in the project. Currently there are multiple commented code lines are present in your controllers. Commented code make it hard to read the code.

Comment (arslanbisharat) [reply README.md:8]: - [ ] [OPTIONAL] please make sure all the changes required in the project is present in your Pull Requests commits. This helps Code Reviwer provide best code review possible l.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @SaidRasinlic, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (SaidRasinlic) [reply README.md:8]: **Thank you! All changes are successfully resolved ✔️ 💯**

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @SaidRasinlic, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**